lib/evss/disability_compensation_form/configuration_rating_info.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    # Configuration for the findRatingInfoPID service
+    #
+    class ConfigurationRatingInfo < EVSS::DisabilityCompensationForm::Configuration
+      API_VERSION = Settings.evss.versions.common
+
+      # :nocov:
```

Comment:
This should get executed when you've got real specs, right? Anything we can do to remove the coverage exemption would make this less worrisome

Reply:
I would echo JP's thought. If we have a :nocov we should have a comment explaining why. The next dev to see this might wrap other code in a :nocov without understanding the why

Reply:
Thank you @johnpaulashenfelter and @charleystran, I meant to take that out.